sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawContainer.java

```diff
@@ -15,7 +16,12 @@ private RawContainer(
             @JsonProperty("image-name") String imageName,
             @JsonProperty("rlimits") WriteOnceLinkedHashMap<String, RawRLimit> rlimits) {
         this.imageName = imageName;
-        this.rlimits = rlimits;
+
+        if (rlimits != null) {
```

Comment:
Jackson will auto-translate missing fields to empty collections if they're a `Collection`, but a `HashMap` is not a collection, which is why we need to set it here explicitly for consistency.